draft-ietf-quic-qcram.md

```diff
@@ -248,24 +440,21 @@ entries in the dynamic table SHOULD be avoided.  When one of the oldest entries
 in the table is still actively used for references, the encoder SHOULD emit an
 Indexed-Duplicate representation instead (see {{indexed-duplicate}}).
 
-## Refreshing Entries with Duplication {#indexed-duplicate}
-
-~~~~~~~~~~  drawing
-    0 1 2 3 4 5 6 7
-   +-+-+-+-+-+-+-+-+
-   |0|0|1|Index(5+)|
-   +-+-+-+---------+
-~~~~~~~~~~
-{:#fig-index-with-duplication title="Indexed Header Field with Duplication"}
-
-*Indexed-Duplicates* insert a new entry into the dynamic table which duplicates
-an existing entry. {{RFC7541}} allows duplicate HPACK table entries, that is
-entries that have the same name and value.
+## Blocked Decoding
 
-This replaces the HPACK instruction for Dynamic Table Size Update (see Section
-6.3 of {{RFC7541}}, which is not supported by HTTP over QUIC.
+For header blocks encoded in non-blocking mode, the encoder needs to forego
+indexed representations that refer to vulnerable entries (see
+{{overview-hol-avoidance}}).  An implementation could extend the header table
```

Comment:
This notion of vulnerable entries is unnecessarily obfuscation in this design.  This is simpler than this: an encoder can't use an insertion until it is acknowledged with HEADER_ACK.

The predecessor QCRAM might have needed this indirection because it coupled different streams together in complicated ways.  This is no longer true.